solutions/bio1/NumberLadder/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { SCodeBlock, SText, STitle, Link } from "components";
+import { brace } from "solutions/util";
+
+export const NumberLadder = () => {
+    return (
+        <>
+            <STitle>Idea</STitle>
+            <SText>
+                Since this problem is asking for the shortest number
+                ladder between two numbers, it is equivalent
+                to finding the shortest path between two nodes in an
+                undirected, unweighted graph. Therefore, we can use
+                breadth-first search (BFS) to solve this problem.
+                However, there is a small catch. Notice that
+                from each number, we need to try to move to
+                all other possible numbers. There are $999$ possibilities
+                for each node, so the runtime of this will blow up very fast.
```

Comment:
I think the phrase "blow up very fast" is a bit misleading. To me, that phrase implies some sort of exponential complexity where the runtime is increasing super quickly, whereas in this case, it seems that it's just a constant factor issue.

Maybe you can instead say, since checking whether an edge exists between two pairs of numbers is an expensive operation, we want to minimise the number of times we do this. Since we need to run a BFS three times and since the graph is undirected (i.e., we'd need to check each edge $6$ times in the worst case), it's better to precalculate the graph to save us from running this expensive operation on the same pair of inputs multiple times.